blockstream/README.md

```diff
@@ -0,0 +1,98 @@
+# Chainlink External Adapter for blockstream
+
+### Environment Variables
+
+| Required? |  Name   | Description | Options | Defaults to |
+| :-------: | :-----: | :---------: | :-----: | :---------: |
+|         | API_KEY |             |         |             |
```

Comment:
Is not required?

Reply:
Doesn't seem like it? The EA uses this endpoint: https://blockstream.info/api/blocks which is publicly accessible

Reply:
Yeah it's not required.